debian/control

```diff
@@ -4,12 +4,21 @@ Priority: optional
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Build-Depends: debhelper (>= 9), libseccomp-dev, libapparmor-dev, libudev-dev, dh-apparmor, indent
 Standards-Version: 3.9.6
-Vcs-Bzr: lp:~snappy-dev/ubuntu-core-launcher/trunk
+Vcs-Git: https://github.com/ubuntu-core/snap-run
 
-Package: ubuntu-core-launcher
+Package: snap-run
 Architecture: any
+Breaks: ubuntu-core-launcher (<< 2.0)
+Replaces: ubuntu-core-launcher (<< 2.0)
 Depends: ${misc:Depends}, ${shlibs:Depends}, apparmor
-Breaks: ubuntu-snappy-cli (<< 1.9), snapd (<< 1.9)
+Description: Launcher for snappy apps
+ This package contains the launcher for applications packaged as snaps
+
+Package: ubuntu-core-launcher
+Architecture: all
```

Comment:
You could add `Section: oldlibs` as a further (informal) clue that this is a transitional thing.